rascaline/src/labels/samples/long_range.rs

```diff
@@ -31,7 +31,7 @@ impl SamplesBuilder for LongRangeSamplesPerAtom {
         for (system_i, system) in systems.iter_mut().enumerate() {
             let species = system.species()?;
 
-            // we want to take all centers matching `species_center` iff
+            // we want to take all centers matching `species_center` if
```

Comment:
`iff` means `if and only if`, if it is not clear we can expand to that!

Reply:
Ahh no it is fine. I just saw the two ff and assuming it is was a typo.